doozerlib/operator_metadata.py

```diff
@@ -260,10 +258,12 @@ def create_metadata_dockerfile(self):
         )
         metadata_dockerfile.labels['com.redhat.delivery.appregistry'] = 'true'
         metadata_dockerfile.labels['name'] = 'openshift/ose-{}'.format(self.metadata_name)
-        try:
-            del(metadata_dockerfile.labels['release'])
-        except KeyError:
-            pass
+        # mangle version according to spec
+        metadata_dockerfile.labels['version'] = '{}-{}.{}'.format(
```

Comment:
... have you tried running a build with this? I didn't think you could have a `-` in that label. I expected it to be: 
```suggestion
        metadata_dockerfile.labels['version'] = '{}.{}.{}'.format(
```